types/superagent/index.d.ts

```diff
@@ -103,14 +107,15 @@ declare namespace request {
     interface Request extends Promise<Response> {
         abort(): void;
         accept(type: string): this;
-        attach(field: string, file: Blob | Buffer | fs.ReadStream | string, filename?: string): this;
+        attach(field: string, file: MultipartValueSingle, filename?: string): this;
```

Comment:
Could you please change `filename?: string` to `options?: string | {filename?: string, contentType?: string}`? See http://visionmedia.github.io/superagent/#multipart-requests